app/src/main/res/values-hu/strings.xml

```diff
@@ -0,0 +1,273 @@
+    <string name="error_artist_name_empty">Az előadó neve nem lehet üres.</string>
+
+    <string name="dialog_title_duplicate_artist">Duplikált előadók</string>
+    <string name="dialog_msg_duplicate_artist">%1$s előadó már létezik.</string>
+
+    <string name="dialog_title_choose_playlist">Válassz lejátszási listát</string>
+
+    <string name="dialog_title_edit_playlist">Lejátszólista szerkesztés</string>
+
+    <string name="dialog_title_choose_backup_content">Válasszon biztonsági másolatot</string>
+    <string name="dialog_title_choose_restore_content">Válasszon visszaállítható tartalmat</string>
+    <string name="choice_preferences">Preferenciák</string>
+    <string name="choice_database">Adatbázis</string>
+    <string name="choice_downloaded_songs">Letöltött dalok</string>
+    <string name="message_backup_create_success">A bizt.mentés sikeresen létrehozva</string>
+    <string name="message_backup_create_failed">Nem sikerült biztonsági mentést készíteni</string>
+    <string name="message_restore_failed">Nem sikerült visszaállítani a biztonsági másolatot</string>
+
+    <!-- Notification -->
+    <string name="channel_name_playback">Zenelejátszó</string>
+    <string name="channel_name_download">Letöltés</string>
+
+    <!-- Noun -->
+    <plurals name="song_count">
+        <item quantity="one">%d dal</item>
+        <item quantity="other">%d dal</item>
+    </plurals>
+    <plurals name="artist_count">
+        <item quantity="one">%d előadó</item>
+        <item quantity="other">%d előadó</item>
+    </plurals>
+    <plurals name="album_count">
+        <item quantity="one">%d album</item>
+        <item quantity="other">%d album</item>
+    </plurals>
+    <plurals name="playlist_count">
+        <item quantity="one">%d lejátszólista</item>
+        <item quantity="other">%d lejátszólista</item>
+    </plurals>
+
+    <!-- Button -->
+    <string name="btn_retry">Újra</string>
+    <string name="btn_play">Lejátszás</string>
+    <string name="btn_play_all">Mind lejátszása</string>
+    <string name="btn_radio">Rádió</string>
+    <string name="btn_shuffle">Keverés</string>
+    <string name="btn_copy_stacktrace">Veremkövetés másolása</string>
+    <string name="btn_report">Jelentés</string>
+    <string name="btn_report_on_github">Jelentés GitHub-on</string>
+
+    <!-- Sort Menu -->
+    <string name="sort_by_create_date">Dátum hozzáadva</string>
+    <string name="sort_by_name">Név</string>
+    <string name="sort_by_artist">Előadó</string>
+    <string name="sort_by_year">Év</string>
+    <string name="sort_by_song_count">Dal szám</string>
+    <string name="sort_by_length">Hossz</string>
+    <string name="sort_by_play_time">Lejátszási idő</string>
+
+    <!-- Snackbar -->
+    <plurals name="snackbar_delete_song">
+        <item quantity="one">%d dal törölve.</item>
+        <item quantity="other">%d dal törölve.</item>
+    </plurals>
+    <plurals name="n_selected">

```

Comment:
For locale "hu" (Hungarian) the following quantity should also be defined: one (e.g. "A kosár tartalma")

Reply:
> are you trying to teach me Hungarian

No, I don't know Hungarian. I'm just copying the error from the build output for you.😅
If no matter what `%d` is, the translation is the same, then just add this line to below:
`<item quantity="one">%d kijelölve</item>`